README.md

```diff
@@ -17,29 +17,38 @@ This project provides a docker container definition that allows deployment of a
 
 1.	Clone https://github.com/negativeeddy/VADeployer
 2.	Open the cloned folder in Visual Studio Code 
-3.	Click Reopen in Container when this pops up 
+3.	Click "Reopen in Container" when the below pop up appears in the bottom right corner of the screen. 
    
    ![Container prompt](docs/CodeReopenInContainer.png)
- 
+   
+    If this pop up doesn't appear, check to see if your have the correct "Shared Drive" is selected.
```

Comment:
the popup not appearing is because of the missing "remote - container" VS Code plugin. not the shared drive (that is a Docker issue)